integrations/cassandra/vendor/vendor.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Why do we have vendor under cassandra folder? Is there any reason for that? Before we had as: `infra-integrations/vendor/vendor.json`
Also all packages are added under `integration/cassandra/vendor` and before it was under: `infra-integrations/vendor/`.  And maybe we could just add `govendor sync` into `Makefile` and get rid of all this files?

Reply:
We don't want to update the rest integrations atm, so we need to vendorize cassandra on isolation.

Reply:
ok, and what about adding just vendor.json and then updating Makefile with `govendor sync`?